src/vm/ecalllist.h

```diff
@@ -548,9 +548,9 @@ FCFuncStart(gRuntimeMethodHandle)
     FCFuncElement("GetMethodBody", RuntimeMethodHandle::GetMethodBody)    
 #ifndef FEATURE_CORECLR
     FCFuncElement("_IsTokenSecurityTransparent", RuntimeMethodHandle::IsTokenSecurityTransparent)
+#endif // FEATURE_CORECLR
```

Comment:
This change should be undone - the managed side is not calling the FCalls anymore